code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -74,3 +74,4 @@ var/global/default_martial_art = new/datum/martial_art
 	var/tail // Name of tail image in species effects icon file.
 
 	var/list/splinted_limbs = list() //limbs we know are splinted
+	emote_types = list("human", "living", "synth", "mob")

```

Comment:
They wouldn't need the `synth` path here if emotes just handled who can use them on their own instead of leaving it up to the mob to define the snowflake categories.

Also, something that just occured to me looking at this - admins can't add emotes to people because of how this mess is setup.